### DISCUSSION TITLE
The option to disable the name of the player that is spectated

### DISCUSSION THREAD

**Discussion by ThanksDawn** [Alf21/TTT2#1286] [Role Ideas]
If someone dies, he will spectated, then he will see the corresponding identity of everyone, and when he is revived, he will go directly to kill the enemy. The existing HUD provides a way to hide your identity, but that takes away the fun of watching, because everyone wants to see the traitor or other bad guys play. The best solution is to remove the name tags on HUD both above the player's model, and only show the identity.

**Comment by CosmoZzZzZzZ** [comment]
Totally agree.

**Comment by HeartlessKillerDog** [comment]
frog good

**Comment by Agutr1** [comment]
Good idea,but it needs you to limit the conditions that reviving players,3 seconds is too fast and may make it harder for traitors to survive.

**Comment by freeang** [comment]
nice frog
